Submitted By: Joe Ciccone <jciccone@gmail.com>
Date: 2008-11-08
Origin: Joe Ciccone
Initial Package Version: 1.8.2
Upstream Status: Unknown
Descriprion: Make busybox play nice with sanatized kernel headers

diff -Naur busybox-1.12.1.orig/networking/libiproute/iptunnel.c busybox-1.12.1/networking/libiproute/iptunnel.c
--- busybox-1.12.1.orig/networking/libiproute/iptunnel.c	2008-09-28 14:04:10.000000000 -0400
+++ busybox-1.12.1/networking/libiproute/iptunnel.c	2008-11-08 11:07:07.000000000 -0500
@@ -14,9 +14,8 @@
  * Phil Karn <karn@ka9q.ampr.org>	990408:	"pmtudisc" flag
  */
 
-#include <netinet/ip.h>
-#include <net/if.h>
-#include <net/if_arp.h>
+#include <linux/if.h>
+#include <linux/if_arp.h>
 #include <asm/types.h>
 #ifndef __constant_htons
 #define __constant_htons htons
